src/Initializer/Template/TYPO3Template.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/* (c) Anton Medvedev <anton@medv.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Deployer\Initializer\Template;
+
+/**
+ * Generate a common (base) deployer configuration
+ *
+ * @author Vitaliy Zhuk <zhuk2205@gmail.com>
+ * @author Anton Medvedev <anton@medv.io>
+ * @codeCoverageIgnore
+ */
+class TYPO3Template extends Template
+{
+    /**
+     * Get content of template.
+     *
+     * @param array $params
+     *
+     * @return string
+     */
+    protected function getTemplateContent($params)
```

Comment:
All framework recipes uses it. So it can be more easily maintained. I don't see any notable changes in your template.